recipes/ozzy-pic/conda_build_config.yaml

```diff
@@ -1,2 +1 @@
-python_min:
-  - 3.10
\ No newline at end of file
+{% set python_min = python_min|default("3.10") %}
```

Comment:
Sorry, I was not clear, what you had here was correct for the conda_build_config.yaml. If you want to get rid of this file you should add that line at the top of the recipe.yaml file.

Reply:
Hi! Thanks for the suggestions, first of all. No worries, I figured it out!